### ISSUE TITLE
REQUEST: Automatic fast forward of load times in FDS games

### ISSUE THREAD

**Issue by lonkelle** [TASEmulators/fceux#478]
Mesen detects massive reads and speeds up all emulation till the operation of I/O of the "floppy" FDS disk is over which greatly reduces FDS load speeds. Here is a link to explain what it technically does for FDS games: https://www.mesen.ca/docs/configuration/preferences.html#fds-settings

**Comment by zeromus** [comment]
Seems like pressing fast forward during this time would be a practical workaround.....

**Comment by lonkelle** [comment]
Yes, of course,  I definitely do that already. This ticket is a quality of life request about automating that manual process just like Mesen does. 

And you could argue it's overkill as Mesen is the only emulator that has this feature. But then my question becomes - why shouldn't FCEUX be as great as Mesen (or better)?

**Comment by zeromus** [comment]
Because fceux is an old man of emulators, he has DNA damage and weak muscles and saggy skin and his joints arent put together very well and it's just better to invest energy in the next generation than try and keep training this codger to do more stuff.

**Comment by lonkelle** [comment]
That’s 100% fair, would you be willing to merge such a feature if I submit a PR for it? FCEUX is my favorite NES emu core so I’d honestly love to contribute.

**Comment by zeromus** [comment]
maybe, maybe not. it depends on how gross it is. fceux is very old and gross already and doesnt take well to having more gross stuff jammed into it. every time that happens it gets harder to jam in the next gross thing which might be more useful. actually making this work in an emulator is waaaaaaay harder than the very simple explanation of how it's done. I dare say impossible. So I advise you not to waste your time. But if you somehow manage to achieve it and it's not too gross... sure.
